src/RWGltf/RWGltf_GltfJsonParser.hxx

```diff
@@ -297,17 +297,19 @@ protected:
   void bindNodeShape (TopoDS_Shape& theShape,
                       const TopLoc_Location& theLoc,
                       const TCollection_AsciiString& theNodeId,
-                      const RWGltf_JsonValue* theUserName)
+                      const RWGltf_JsonValue* theUserName,
```

Comment:
I would suggest to use `const Handle(TDataStd_NamedData)&`.
Originally input parameter must be not updated. It is originally a part of the new internal class.
The same for 2 more new methods in current file